canonical/ubuntu.com
# Refresh /server/hyperscale

*this will act as the feature branch for the server updates*

## Done

- Refresh /server/hyperscale based on [copydoc](https://docs.google.com/document/d/1f56CCzH519lIVa4KvxDt0QpxYFMK-bMdVlKo4TvPSIk/edit) and [design](https://www.figma.com/file/Kw2SwMkH49tN54NbnKs5dt/U.com---server-(rebranding)?type=design&node-id=1-10&mode=design&t=ozrDP1wwHOGlBNCq-0)
- I used [this](https://docs.google.com/document/d/1kq0SPv-PAIfL9nHGxgNDpHhr5OplE2iNaZZiqG4RT48/edit#heading=h.qr5jy1sbfmj) as reference for the contextual footer

## QA

- Check [the demo](https://ubuntu-com-13521.demos.haus/server/hyperscale) matches the [copydoc](https://docs.google.com/document/d/1f56CCzH519lIVa4KvxDt0QpxYFMK-bMdVlKo4TvPSIk/edit) and design [design](https://docs.google.com/document/d/1f56CCzH519lIVa4KvxDt0QpxYFMK-bMdVlKo4TvPSIk/edit)
- Test on desktop and mobile views

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-8256
